src/main/java/io/github/thebusybiscuit/slimefun4/api/ErrorReport.java

```diff
@@ -84,6 +86,8 @@ public ErrorReport(T throwable, SlimefunAddon addon, Consumer<PrintStream> print
     @ParametersAreNonnullByDefault
     public ErrorReport(T throwable, Location l, SlimefunItem item) {
         this(throwable, item.getAddon(), stream -> {
+
+            stream.println("Error Generated: " + dateFormat.format(LocalDateTime.now()));
```

Comment:
This should not be location-based but rather added to **every** error-report in my opinion. So please move that down to the part where Java version etc... are added as well